AirCasting/CreateSessionViews/ChooseCustomLocationView.swift

```diff
@@ -7,6 +7,7 @@ import SwiftUI
 struct ChooseCustomLocationView: View {
     @State private var isConfirmCreatingSessionActive: Bool = false
     @State private var location = ""
+    @State var placePickerDismissed: Bool = false
```

Comment:
Could you rename it to `placePickerPresented`? It feels easier to understand when it's not a negation 🙈